book/src/2.getting-started/1.quickstart.md

```diff
@@ -0,0 +1,100 @@
+# Quickstart
+
+Get started with Teams AI v2 quickly using the Teams CLI.
+
+## Set up a new project
+
+### Prerequisites
+
+- **Node.js** v.20 or higher. Install or upgrade from [nodejs.org](https://nodejs.org/).
+- **Text editor** (e.g., [Visual Studio Code](https://code.visualstudio.com/)).
+- **Terminal** (e.g., Command Prompt, PowerShell, iTerm2, or even VS Code's integrated terminal).
+
+## Instructions
+
+### Install the Teams CLI
+
+Use your terminal to install the Teams CLI globally using npm:
+
+```sh
+npm install -g @microsoft/teams.cli@latest
+```
+
+> ℹ️ **INFO:**
```

Comment:
Consider using alerts instead of custom stuff like this:
https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts. We have support for these in these docs